suricata.yaml.in

```diff
@@ -736,6 +736,7 @@ af-xdp:
 dpdk:
   eal-params:
     proc-type: primary
+    #pci-whitelist: ["01:00.0", "01:00.1"]
```

Comment:
this is not a setting suricata supports, so it should not be in the config

Reply:
Hello, we have been waiting for a long time for suricata to officially support the native configuration of the dpdk interface, and it looks like it will be released soon, and it seems to be complete and working very well. Here, what I want to explain is that the parameter usage of --pci-whitelist is used in my personal usage scenario: ./build/app/testpmd -c 0xf --pci-whitelist 06:00.2 --pci-whitelist 06:00.3 -- -i
Multiple -w/--pci-whitelist arguments usually do not cause dpdk errors and are supported by dpdk itself.
This can shield unnecessary pci interfaces in the system from being exposed to dpdk. My improvement is to expect eal-params to support processing list parameters with the same key. Here just use --pci-whitelist as an example.

2. In my verification, the yaml file cannot support multiple configuration items with the same name, so the pci-whitelist configuration item in the second line in the yaml file will refresh the pci-whitelist configuration item in the first line. So actually, I tried writing two lines of pci-whitelist configuration items in the yaml file, but it failed. This is a compromise that I can think of. Of course, if this configuration item does not conform to the design idea of suricata, it seems that there is no other way to support it for the time being.
Anyway, hope this helps with suricata dpdk. Thank you so much.